INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: POST на http://localhost:8001/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 200 OK от http://localhost:8001/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: POST на http://localhost:8001/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 200 OK от http://localhost:8001/api/v1/authentication/login
INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: POST на http://localhost:8001/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 200 OK от http://localhost:8001/api/v1/users
INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: DELETE на http://localhost:8001/api/v1/users/28ff3052-e7b0-4b0f-a257-6db607070591
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 200 OK от http://localhost:8001/api/v1/users/28ff3052-e7b0-4b0f-a257-6db607070591
INFO     BASE_ASSERTIONS:base.py:24 Проверяем, что статус код ответа сервера соответствует 200
INFO     HTTP_CLIENT:event_hooks.py:24 Отправляем запрос: GET на http://localhost:8001/api/v1/users/28ff3052-e7b0-4b0f-a257-6db607070591
INFO     HTTP_CLIENT:event_hooks.py:28 Получаем ответ: 404 Not Found от http://localhost:8001/api/v1/users/28ff3052-e7b0-4b0f-a257-6db607070591
INFO     BASE_ASSERTIONS:base.py:24 Проверяем, что статус код ответа сервера соответствует 404
INFO     USERS_ASSERTIONS:users.py:230 Проверяем ответ сервера на запрос несуществующего пользователя
INFO     ERRORS_ASSSERTIONS:errors.py:75 Проверяем ответ сервера с ожидаемой внутренней ошибкой
INFO     BASE_ASSERTIONS:base.py:45 Проверяем, что значение details соответствует ожидаемому User not found
INFO     SCHEMA_ASSERTIONS:schema.py:26 Проверяем, соответствует ли ответ сервера JSON-схеме